x-pack/test_serverless/api_integration/test_suites/common/console/proxy_route.ts

```diff
@@ -7,38 +7,31 @@
 
 import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
 
 export default function ({ getService }: FtrProviderContext) {
-  const supertest = getService('supertest');
   const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  let roleAuthc: RoleCredentials;
 
   describe('POST /api/console/proxy', () => {
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+    });
+    after(async () => {
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
+    });
     describe('system indices behavior', () => {
-      it('returns warning header when making requests to .kibana index', async () => {
```

Comment:
Why do we need to remove this test? Seems it's needed to assert the warning header existence when pinging .kibana

Reply:
https://github.com/elastic/kibana/pull/181741#discussion_r1596685020